library/kani/src/lib.rs

```diff
@@ -39,7 +53,10 @@ pub use invariant::Invariant;
 /// ```
 #[inline(never)]
 #[rustc_diagnostic_item = "KaniAssume"]
-pub fn assume(_cond: bool) {}
+pub fn assume(_cond: bool) {
+    #[cfg(feature = "exe_trace")]
```

Comment:
What's the reasoning behind this change? From what I see, it allows for easier IDE checking (i.e., it can check the code inside the if without having to enable feature flags during compilation). But the downside is that it relies on the compiler to remove dead code in conditionals, otherwise it bloats the symbolic expressions.

Reply:
Reason is that it improves code readability.